packages/api/tests/models/test-rules-model.js

```diff
@@ -42,6 +43,17 @@ const onetimeRule = {
   state: 'ENABLED'
 };
 
```

Comment:
As we've duplicated logic in L46-56 in the migration unit test we should to move getKinesisEventMappings to a test helper.